tests/next_tests/integration_tests/cases.py

```diff
@@ -490,11 +490,34 @@ def unstructured_case(
     mesh_descriptor,
     exec_alloc_descriptor: test_definitions.EmbeddedDummyBackend | next_backend.Backend,
 ):
+    ditc_3d = mesh_descriptor.offset_provider
+    ditc_3d["Koff"] = KDim
     yield Case(
         None
         if isinstance(exec_alloc_descriptor, test_definitions.EmbeddedDummyBackend)
         else exec_alloc_descriptor,
         offset_provider=mesh_descriptor.offset_provider,
+        default_sizes={
+            Vertex: mesh_descriptor.num_vertices,
+            Edge: mesh_descriptor.num_edges,
+            Cell: mesh_descriptor.num_cells,
+        },
+        grid_type=common.GridType.UNSTRUCTURED,
+        allocator=exec_alloc_descriptor.allocator,
+    )
+
+
+@pytest.fixture
+def unstructured_case_3d(
+    mesh_descriptor,
+    exec_alloc_descriptor: test_definitions.EmbeddedDummyBackend | next_backend.Backend,
+):
```

Comment:
```suggestion
@pytest.fixture
def unstructured_case_3d(unstructured_case):
  return dataclasses.replace(unstructured_case, default_sizes={**unstructured_case.default_sizes, KDim: 10}, {**unstructured_case.offset_provider, "KOff": KDim})
```
The idea was to make this fixture build upon the `unstructured_case` (avoiding code duplication).